dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -140,6 +149,11 @@ public abstract class Entry<S extends Entry, T extends Version> implements Compa
     @ApiModelProperty(value = "Implementation specific ID for the container in this web service", position = 0)
     private long id;
 
+    @JsonProperty("type")
```

Comment:
Goes back to that linked thread, I'm not sure why I needed to create this property "manually" and why it needs to be set as well. It looks the Jackson annotations above should just work, but they do not.

Reply:
See https://www.baeldung.com/jackson-annotations#bd-jackson-polymorphic-type-handling-annotations (i.e. it should work without needing to declare this, but it doesn't)